Backend/api/views/transaction_view.py

```diff
@@ -189,5 +190,81 @@ def get(self, request, transaction_id):
                 "broadcasted": transaction.is_broadcasted,
             }
         })
+
+class CreatePSBT(APIView):
```

Comment:
I thought this view should be in psb_transaction_view

Reply:
Oh, it is. I forgot to remove it from here,will do that now